DirectProgramming/DPC++FPGA/Tutorials/Features/pipes/README.md

```diff
@@ -1,309 +1,309 @@
-
-Navigate to the "System Viewer" to visualize the structure of the kernel system. Identify the pipe connecting the two kernels.
-
-## Running the Sample
-
- 1. Run the sample on the FPGA emulator (the kernel executes on the CPU):
-     ```
-     ./pipes.fpga_emu     (Linux)
-     pipes.fpga_emu.exe   (Windows)
-     ```
-2. Run the sample on the FPGA device:
-     ```
-     ./pipes.fpga         (Linux)
-     ```
-
-### Example of Output
-You should see the following output in the console:
-
-1. When running on the FPGA emulator
-    ```
-    Input Array Size: 8192
-    Enqueuing producer...
-    Enqueuing consumer...
-
-    Profiling Info
-      Producer:
-        Start time: 0 ms
-        End time: +8.18174 ms
-        Kernel Duration: 8.18174 ms
-      Consumer:
-        Start time: +7.05307 ms
-        End time: +8.18231 ms
-        Kernel Duration: 1.12924 ms
-      Design Duration: 8.18231 ms
-      Design Throughput: 4.00474 MB/s
-
-    PASSED: The results are correct
-    ```
-    NOTE: The FPGA emulator does not accurately represent the performance nor the kernels' relative timing (i.e., the start and end times).
-
-2. When running on the FPGA device
-    ```
-    Input Array Size: 1048576
-    Enqueuing producer...
-    Enqueuing consumer...
-
-    Profiling Info
-      Producer:
-        Start time: 0 ms
-        End time: +4.481 ms
-        Kernel Duration: 4.481 ms
-      Consumer:
-        Start time: +0.917 ms
-        End time: +4.484 ms
-        Kernel Duration: 3.568 ms
-      Design Duration: 4.484 ms
-      Design Throughput: 935.348 MB/s
-
-    PASSED: The results are correct
-    ```
+# Data Transfers Using Pipes
+This FPGA tutorial shows how to use pipes to transfer data between kernels.
+
+***Documentation***:  The [DPC++ FPGA Code Samples Guide](https://software.intel.com/content/www/us/en/develop/articles/explore-dpcpp-through-intel-fpga-code-samples.html) helps you to navigate the samples and build your knowledge of DPC++ for FPGA. <br>
+The [oneAPI DPC++ FPGA Optimization Guide](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide) is the reference manual for targeting FPGAs through DPC++. <br>
+The [oneAPI Programming Guide](https://software.intel.com/en-us/oneapi-programming-guide) is a general resource for target-independent DPC++ programming.
+
+| Optimized for                     | Description
+---                                 |---
+| OS                                | Linux* Ubuntu* 18.04/20.04, RHEL*/CentOS* 8, SUSE* 15; Windows* 10
+| Hardware                          | Intel&reg;reg; Programmable Acceleration Card (PAC) with Intel A&reg;ria&reg; 10 GX FPGA <b&reg;> Intel&reg; FPGA Programmable Acceleration Card (PAC) D5005 (with I&reg;tel Stratix&reg; 1&reg; SX) <br> Intel&reg; FPGA 3rd party / custom platforms with oneAPI support <&reg;r> *__Note__: Intel&reg; FPGA PAC hardware is only compatible with Ubuntu 18.04* 

```

Comment:
This looks wrong.  In fact this whole file looks really wrong.

I don't understand how this could have happened, weren't all these changes done with a script?  This makes me nervous about this entire PR, I don't want to go through 90 files with a fine toothed comb.